tests/test_classical_mis.py

```diff
@@ -1,13 +1,71 @@
 from typing import Callable
 import networkx as nx
 import pytest
+from pathlib import Path
 
 # Define classical solver configuration
 from mis.solver.solver import MISInstance, MISSolver
 from mis.pipeline.config import SolverConfig
 from mis.shared.types import MethodType
 from mis.pipeline.kernelization import Kernelization
 
+TEST_FILES_DIR = Path.cwd() / "tests/test_files"
+
+
+@pytest.mark.parametrize("dimacs_to_nx", [TEST_FILES_DIR / "a265032_1tc.32.txt"], indirect=True)
+@pytest.mark.parametrize("preprocessor", [None, lambda graph: Kernelization(graph)])
+def test_for_dimacs_32_node_graph(
+    dimacs_to_nx: nx.Graph, preprocessor: None | Callable[[nx.Graph], Kernelization]
+) -> None:
+    """
+    Classical MIS solver for a standard graph benchmark dataset in DIMACS format.
+
+    Can be found here: https://oeis.org/A265032/a265032.html
+    """
+    graph = dimacs_to_nx
+    assert graph.number_of_nodes() == 32
+    assert graph.number_of_edges() == 68
+
+    config = SolverConfig(method=MethodType.EAGER, max_iterations=1, preprocessor=None)
+
+    # Create the MIS instance
+    instance = MISInstance(graph)
+
+    # Run the solver
+    solver = MISSolver(instance, config)
+    solutions = solver.solve().result()
+
+    assert len(solutions[0].nodes) == 12
```

Comment:
Since the solver is non-deterministic, this may not always work.

Reply:
Yeah, marking them as flaky may be a solution.

I don't understand what you mean by "strictly compare". 

Can we at least check that the response is always an independent set? Ideally a maximal independent set?

Reply:
> Should we aim to return a set of identical node labels as well as the size ?

If it's the same size, I think we're good, if it's really a MIS.

> Is there a way to check this ?

There are methods in `Kernelization`, I believe.